code/datums/datacore.dm

```diff
@@ -252,7 +251,7 @@ var/global/list/PDA_Manifest = list()
 		foundrecord.fields["real_rank"] = real_title
 
 var/record_id_num = 1001
-/datum/datacore/proc/manifest_inject(mob/living/carbon/human/H)
+/datum/datacore/proc/manifest_inject(mob/living/carbon/human/H, client/C)

```

Comment:
Oh, I wasn't suggesting adding this argument. our `manifest_inject` is completely different.